### ISSUE TITLE
Focus state not activated on keyboard focus for ActionCard component

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#4230]
**Describe the bug**

Our focus states are only being activated on mouse hover for when using the ActionCard component. They should also be activated on keyboard focus.

<img width="1470" alt="Screenshot 2021-10-14 at 12 02 28" src="https://user-images.githubusercontent.com/62268199/137305413-8918590c-aa4a-4689-9e9b-cf47067fb98b.png">

**To Reproduce**

1. Go to '[ethereum.org](https://ethereum.org/en/)'
2. Tab to 'Get started section'
3. Tab onto the 'Pick a wallet' ActionCard link

**Expected behavior**

Focus state should be activated on keyboard focus as well as on mouse hover.

**Screenshots**

[Here is a GIF of the bug](https://i.gyazo.com/211529313bb70ddeeae0f730f14b1984.gif)

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.
